std_msgs/Header header
uint32 RequestId
sensor_msgs/PointCloud mouseClicks
